packages/material-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -73,21 +74,15 @@ const CheckboxesWidget = ({
               onChange={_onChange(option)}
               onBlur={_onBlur}
               onFocus={_onFocus}
+              {...uiProps}
             />
           );
-          return inline ? (
-            <FormControlLabel
+          
+          return <FormControlLabel
               control={checkbox}
               key={index}
               label={option.label}
```

Comment:
Is this missing the `|| schema.title` that was added in all the other places on purpose?

Reply:
No, you can find this in line 62 - it's part of the `FormLabel`, not as part of the FormControlLabel